AIMER-ROOT/templates/layout/partials/scripts.html

```
@@ -1,15 +1,10 @@
 {% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)